website/docs/usage/linguistic-features.mdx

```diff
@@ -113,7 +113,7 @@ print(doc[2].morph)  # 'Case=Nom|Person=2|PronType=Prs'
 print(doc[2].pos_)  # 'PRON'
 ```
 
-## Lemmatization {id="lemmatization",model="lemmatizer",version="3"}
+## Lemmatization {id="lemmatization",version="3"}
```

Comment:
If you're going to remove `model="lemmatizer"` here, then I think it makes sense to also add it to the rule-based lemmatizer section?

Reply:
The rule-based lemmatizers all require at least POS from a model?

Reply:
Ok, so for the rule-based section you want to add `model="tagger"`?

Reply:
Hmm, there's not a simple description here that applies across the board. I guess `morphologizer` is accurate enough for most cases is the closest.